tests/utils_test.py

```diff
@@ -390,11 +390,16 @@ def test_construct_daynames():
         'DTEND;TZID=Europe/Berlin;VALUE=DATE-TIME:20131026T000000')),
 
     # start and end date same, explicit end date (but no year) given
-    # XXX FIXME: if no explicit year is given for the end, this_year is used
     ('25.10.2013 18:00 26.10. 20:00 Äwesöme Event',
      _create_vevent(
         'DTSTART;TZID=Europe/Berlin;VALUE=DATE-TIME:20131025T180000',
-        'DTEND;TZID=Europe/Berlin;VALUE=DATE-TIME:20131026T200000')),
+        'DTEND;TZID=Europe/Berlin;VALUE=DATE-TIME:20141026T200000')),
```

Comment:
I don't think this should generate an event lasting for more than a year (and the FIXME is outdated any way).